flocker/testtools/__init__.py

```diff
@@ -592,11 +596,11 @@ def build(self, dockerfile_variables=None):
             b'--tag=%s' % (tag,),
             docker_dir.path
         ]
-        check_call(command)
+        run_process(command)
         # XXX until https://clusterhq.atlassian.net/browse/FLOC-409 is
```

Comment:
This issue is fixed, so this comment should be removed or changed (in a follow-up if you prefer).